control.c

```diff
@@ -44,7 +44,7 @@ static ssize_t control_read(struct file *file,
 {
     int len;
     static const char *str = "Virtual V4L2 compatible camera device\n";
-    pr_debug("read %ld %dB\n", (long) buffer, (int) length);
+    pr_debug("read %ld %dB\n", (long __force) buffer, (int) length);
```

Comment:
Maybe it is not necessary to dump the address of buffer. How do you think?

Reply:
Yes, conversion type is unnecessary here.